src/components/Pagination/Pagination-story.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+
+import {
+  withKnobs,
+  array,
+  boolean,
+  number,
+  text,
+} from '@storybook/addon-knobs';
+import Pagination from '../Pagination';
+import { componentsX } from '../../internal/FeatureFlags';
+
+const props = () => ({
+  disabled: boolean('Disable backward/forward buttons (disabled)', false),
+  page: number('The current page (page)', 1),
+  totalItems: number('Total number of items (totalItems)', 103),
+  pagesUnknown: boolean('Total number of items unknown (pagesUnknown)', false),
+  pageInputDisabled: boolean('Disable page input (pageInputDisabled)', false),
+  isLastPage: componentsX
+    ? null
+    : boolean('At the last page (isLastPage)', false),
+  backwardText: text(
+    'The description for the backward icon (backwardText)',
+    'Previous page'
+  ),
+  forwardText: text(
+    'The description for the backward icon (forwardText)',
+    'Next page'
+  ),
+  pageSize: number('Number of items per page (pageSize)', 10),
+  pageSizes: array('Choices of `pageSize` (pageSizes)', [10, 20, 30, 40, 50]),
+  itemsPerPageText: text(
+    'Label for `pageSizes` select UI (itemsPerPageText)',
+    'Items per page:'
+  ),
+  onChange: action('onChange'),
+});
+
+storiesOf('Pagination', module)
+  .addDecorator(withKnobs)
+  .addDecorator(story => <div style={{ width: '800px' }}>{story()}</div>)
+  .add('Pagination', () => <Pagination {...props()} />, {
+    info: {
+      text: `
+            The pagination component is used to switch through multiple pages of items, when only a maxium number of items can be displayed per page. Can be used in combination with other components like DataTable.
+          `,
+    },
+  })
+  .add(
+    '↪︎ multipe Pagination components',
```

Comment:
as much as I love a good unicode character, I think the sidebar might get cluttered if we add these willy nilly.
```
              ,,))))))));,
           __)))))))))))))),
\|/       -\(((((''''((((((((.
-*-==//////((''  .     `)))))),
/|\      ))| o    ;-.    '(((((                                  ,(,
         ( `|    /  )    ;))))'                               ,_))^;(~
            |   |   |   ,))((((_     _____------~~~-.        %,;(;(>';'~
            o_);   ;    )))(((` ~---~  `::           \      %%~~)(v;(`('~
                  ;    ''''````         `:       `:::|\,__,%%    );`'; ~
                 |   _                )     /      `:|`----'     `-'
           ______/\/~    |                 /        /
         /~;;.____/;;'  /          ___--,-(   `;;;/
        / //  _;______;'------~~~~~    /;;/\    /
       //  | |                        / ;   \;;,\
      (<_  | ;                      /',/-----'  _>
       \_| ||_                     //~;~~~~~~~~~
           `\_|                   (,~~  
                                   \~\
                                    ~~
```

Reply:
@vpicone  lol! thanks. i thought it might be potentially helpful here bc the link name "multiple Pagination components" isn't as clear of a singular list item (especially if it line-wraps). but if u wanna edit i totes don't mind you doing it in another PR since this one is already merged!